types/mssql/index.d.ts

```diff
@@ -340,15 +342,11 @@ export declare class Request extends events.EventEmitter {
     public resume(): boolean;
 }
 
-export declare class RequestError implements Error {
-    constructor(message: string, code?: any)
-    public name: string;
-    public message: string;
-    public code: string;
+export declare class RequestError extends MSSQLError {
     public number?: number;
+    public lineNumber?: number;
     public state?: number;
     public class?: number;
```

Comment:
It seems that `state` and `class` are strings:
https://github.com/tediousjs/node-mssql/blob/2397bbcc4a1c984bd46d2e03306d9fe4c5819d4c/lib/error/request-error.js#L10

I know you did not change this, but it might as well get fixed now that we are at it :-)